code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1439,6 +1439,9 @@ GLOBAL_LIST_EMPTY(roundstart_race_datums)
 //////////////////
 
 /datum/species/proc/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	if(attacker_style && attacker_style.help_act(user,target)) // thank you hippiestation, bless.
+		return 1
```

Comment:
not modular. Needs either to be modular or have //skyrat edit

Reply:
ditto